src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+package mate.academy.dao;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    private static final String TITLE_COLUMN_NAME = "title";
+    private static final String PRICE_COLUMN_NAME = "price";
+    private static final String ID_COLUMN_NAME = "id";
+    private static final String CREATE_QUERY = "INSERT INTO books (title, price) VALUES (?,?)";
+    private static final String FINDBYID_QUERY = "SELECT * FROM books WHERE id = ?";
```

Comment:
```suggestion
    private static final String FIND_BY_ID_QUERY = "SELECT * FROM books WHERE id = ?";
```
increase readability